rma/models/rma_order_line.py

```diff
@@ -622,3 +622,9 @@ def action_view_rma_lines(self):
             result['views'] = [(res and res.id or False, 'form')]
             result['res_id'] = rma_lines[0]
         return result
+
+    @api.constrains("partner_id", "rma_id")
+    def _check_partner_id(self):
+        if self.rma_id and self.partner_id != self.rma_id.partner_id:
+            raise UserError(_(
```

Comment:
```suggestion
            raise ValidationError(_(
```

Reply:
This is another module. Solved here: https://github.com/Eficent/stock-rma/pull/73